src/metadata/v57.json

```diff
@@ -1,4 +1,39 @@
 [
+  {
+    "directoryName": "forecastingFilters",
+    "inFolder": false,
+    "metaFile": false,
+    "suffix": "forecastingFilter",
+    "xmlName": "ForecastingFilter"
+  },
+  {
+    "directoryName": "forecastingFilterConditions",
```

Comment:
Documentation states that folder and suffix start with Uppercase but it is wrong: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_forecastingfiltercondition.htm

Proof:

<img width="642" alt="image" src="https://user-images.githubusercontent.com/28541804/225067098-2d8cc546-7548-4996-86a0-5da0ecac98c0.png">